pkg/ttnpb/udp/translation.go

```diff
@@ -20,6 +20,7 @@ import (
 	"strings"
 	"time"
 
+	pbtypes "github.com/gogo/protobuf/types"
```

Comment:
I thought we only did this if we were also using our own types package

Reply:
I typically do this for my future self and other devs to not rename imports later when using `pkg/types` and creating unnecessary diffs

Reply:
I thought we always use `pbtypes`